src/coreclr/jit/CMakeLists.txt

```
@@ -626,7 +626,11 @@ endif()
 if (CLR_CMAKE_TARGET_WIN32 AND CLR_CMAKE_PGO_INSTRUMENT)
   # Copy PGO dependency to target dir
   set(PGORT_DLL "pgort140.dll")
-  find_path(PGORT_DIR ${PGORT_DLL} REQUIRED)
+  if(ARCH_TARGET_NAME STREQUAL "arm64")
```

nit: quotes are not needed (see line 617). Condition can be simplified to:
```suggestion
  if(CLR_CMAKE_TARGET_ARCH_ARM64)
```